Add support to set a single child builder for several parent builders

natto1784 opened the pull request:
- Implement a simple 'embed' function for 'CreateInteractionResponseData' and 'CreateInteractionResponseFollowup' builders, similar to 'set_embed' for 'CreateMessage' builder
- Implement a simple 'set_action_row' function for 'CreateComponents' builder, similar to 'set_embed' for 'CreateMessage' builder

natto1784 committed:
Add support for setting an embed for interaction response data

Implement a simple 'embed' function for 'CreateInteractionResponseData' and 'CreateInteractionResponseFollowup' builders, similar to 'set_embed' for 'CreateMessage' builder

natto1784 commented:
Also how likely is it that functions can be renamed to be uniform for similar builders? For eg, the function names differ a lot for `CreateInteractionResponseData` when compared to `CreateMessage`, this can cause confusion sometimes.

arqunis commented:
> Also how likely is it that functions can be renamed to be uniform for similar builders? For eg, the function names differ a lot for `CreateInteractionResponseData` when compared to `CreateMessage`, this can cause confusion sometimes.

Under normal circumstances, that kind of change would need to be put on the `next` branch. Fortunately, in the case of interactions, they are allowed to be even on the `current` branch, as code marked under the `unstable_discord_api` feature can disregard the prohibition of breaking changes on `current`. If you want, you can make the methods of interaction builders be consistent with builders like `CreateMessage`.

natto1784 committed:
Add support for setting an action row for components

Implement a simple 'set_action_row' function for 'CreateComponents' builder, similar to 'set_embed' for 'CreateMessage' builder

natto1784 committed:
Rename interaction builder functions to be consistent with other builders

natto1784 committed:
Add some functions to interaction builders

Add functions like add_embeds and set_embed for interaction builders similar to how they are in other builders for consistency
